docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -0,0 +1,696 @@
+    "\n",
+    "sc.plot(solution1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96f30fb7-1ff3-45c3-bb3b-21a23ddd08e6",
+   "metadata": {},
+   "source": [
+    "## Multi-dimensional spatial binning"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ee28604-8ec5-46bf-a9a9-26667eb7f314",
+   "metadata": {},
+   "source": [
+    "`sc.bin` can handle multiple dimensions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97965f28-1bd4-45dd-856a-2b55d0d53894",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = table.coords['x_pos']\n",
+    "y = table.coords['y_pos']\n",
+    "z = table.coords['z_pos']\n",
+    "x_edges = sc.linspace('x_pos', x.min().value, x.max().value, num=32, unit='mm')\n",
+    "y_edges = sc.linspace('y_pos', y.min().value, y.max().value, num=32, unit='mm')\n",
+    "z_edges = sc.linspace('z_pos', z.min().value, z.max().value, num=32, unit='mm')\n",
+    "binned_xyz = sc.bin(table, edges=[z_edges, y_edges, x_edges])\n",
+    "binned_xyz['z_pos', 20:].bins.sum().plot(norm='log', aspect='equal')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "040d88fa-e938-463a-bfbd-d90edd5eb80c",
+   "metadata": {},
+   "source": [
+    "Above we can see a cut through the detector assembly, which has the shape of a thick cylinder mantle.\n",
+    "\n",
+    "The advantage of binned data over histogrammed data is that metadata for each underlying event is still present.\n",
+    "We can therefore change the binning, or bin in additional dimensions.\n",
+    "For example, we can select the slice containing $z = 0$ and turn it into a higher-resolution cut:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2488e51c-db3f-419f-80c7-b3be8cb50d51",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x_fine = sc.linspace('x_pos', x.min().value, x.max().value, num=41, unit='mm')\n",
+    "y_fine = sc.linspace('y_pos', y.min().value, y.max().value, num=101, unit='mm')\n",
+    "z_slice = binned_xyz['z_pos', sc.scalar(0.0, unit='mm')]\n",
+    "xy_cut = sc.bin(z_slice, edges=[y_fine, x_fine])\n",
+    "xy_cut"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "122a7df6-8172-433b-8ac3-4d870389291f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xy_cut.bins.sum().transpose().plot(aspect='equal')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e3a73ba-f911-46e0-b858-2a79cd5f992b",
+   "metadata": {},
+   "source": [
+    "### Exercise 2\n",
+    "\n",
+    "- The detector assembly is cylindrical, aligned with the z-axis.\n",
+    "  Compute the distance from the z-axis (from `x` and `y` defined above) and store it as a new coordinate in `table`.\n",
+    "  This is the radius in cylinder coordinates.\n",
+    "- Define bin edges for the radius.\n",
+    "- Bin `table` by `z_pos` and the radius.\n",
```

Comment:
```suggestion
    "- Bin `table` by `z_edges` and the radius.\n",
```
or
```suggestion
    "- Bin `table` in `z_pos` and the radius.\n",
```

Reply:
Are you sure? I would say "bin the data by wavelength" not "bin the data in wavelength"? We need a native speaker.

Reply:
> We need a native speaker.

Agree.
But I would also say that 'bin the data by wavelength' can refer to either the coord or edges. I think English is unclear in this case. Maybe this can be avoided using `bin the data into bins according to `z_edges`'?

Reply:
I find it confusing to refer to something that is not a property of the data. I want to refer to a column of the metadata, i.e., the event-coord `z_pos`.